index.html

```diff
@@ -280,7 +280,50 @@ <h3>Contact me</h3>
+                  <p class="metaCategories">DEFAULT</p>
+                  <p class="metaSeparators"><img src="./media-library/Counter.png" alt="seperator" width="" height=""></p>
+                  <p class="metaCategories inactiveCategories">DEFAULT</p>
+                  <p class="metaSeparators"><img src="./media-library/Counter.png" alt="seperator" width="" height=""></p>
+                  <p class="metaCategories inactiveCategories">DEFAULT</p>
+                </div>
+              </div>
+              <p onclick="deactivateModalPopup()"><img src="./media-library/modalcloseicon.png" class="modalcloseicon" alt="modalcloseicon" width="" height=""></p>
+            </div>
+       
+
+            <p><img class="workImages" src="./media-library/Snapshoot-Portfolio.png" alt="work image" width="" height=""></p>
+            <div class="modalContents workContents">
+              <p class="modalDescription workDescriptions defaultFontFamily">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentLorem Ipsum is simply dummy text of the printing and typesetting industry. </p>
+              <div class="modalCTAs workCTAs">
+                <ul class="workTags defaultFontFamily">
+                  <li class=""><a href="#"></a></li>
+                  <li class="hideLists" ><a href="#"></a></li>
+                  <li class=""><a href="#"></a></li>
+                  <li class=""><a href="#"></a></li>
+                  <li class="hideLists" ><a href="#"></a></li>
+                </ul>
+            <p class="modalBtnContainer">
+            <a href="#"><button class="modalBtns workBtns defaultFontFamily">See live <img src="./media-library/seeliveicon.png" alt="work image" width="" height=""></button></a>
+            <a href="#"><button class="modalBtns workBtns defaultFontFamily">See source <img src="./media-library/seesourceicon.png" alt="work image" width="" height=""></button></a>
+            </p>
```

Comment:
- [ ] I would kindly suggest you to work a little bit on the see project and see source button, the images icon should be align with each of the text ,just like the fima template ,its good to stick to the figma template to make your website look profesional and user friendly 
- [ ] your design
![lll](https://github.com/geekelo/a-personal-portfolio-site/assets/92458236/4f604e50-669e-4c74-b233-6e6df563a21d)

- [ ] figma design
![bbb](https://github.com/geekelo/a-personal-portfolio-site/assets/92458236/fe6b6141-86ed-49e7-93a7-7eeb22933304)